docs/contributing/contributing-to-airbyte/README.md

```diff
@@ -4,45 +4,92 @@ description: 'We love contributions to Airbyte, big or small.'
 
-We do ask that you read this document and send us any issues or raise pull requests to make sure we understand clearly what you are asking for. This will help ensure we are making the right decisions and taking the right actions.
+However, for those who want a bit more guidance on the best way to contribute to Airbyte, read on. This document will cover what we're looking for. By addressing the points below, the chances that we can quickly merge or address your contributions will increase.
 
-**Note that you are able to create integrations using the language you want, as Airbyte connections run as Docker containers.**
+## Table of Contents
 
-## **Review process**
+1. [Code of conduct](https://docs.airbyte.io/v/dx-to-contribute_suggestion/contributing/contributing-to-airbyte#1-code-of-conduct)
+2. [Airbyte specification](https://docs.airbyte.io/v/dx-to-contribute_suggestion/contributing/contributing-to-airbyte#2-the-airbyte-specification)
+3. [First-time contributors, welcome!](https://docs.airbyte.io/v/dx-to-contribute_suggestion/contributing/contributing-to-airbyte#3-first-time-contributors-welcome)
+4. [Areas for contributing](https://docs.airbyte.io/v/dx-to-contribute_suggestion/contributing/contributing-to-airbyte#4-areas-for-contributing)
+5. [Ways you can contribute](https://docs.airbyte.io/v/dx-to-contribute_suggestion/contributing/contributing-to-airbyte#5-ways-you-can-contribute)
+6. [Review process](https://docs.airbyte.io/v/dx-to-contribute_suggestion/contributing/contributing-to-airbyte#6-review-process)
 
-**‌‌**If you are considering adding to the codebase or contributing a new integration, first a big thank you. We sincerely appreciate your help in our mission to solve the data integration problem!
+## 1. Code of conduct
+
+Please follow our [Code of conduct](https://docs.airbyte.io/contributing/code-of-conduct) in the context of any contributions made to Airbyte.
+
+## 2. The Airbyte specification
+
+Before you can start contributing, you need to understand [Airbyte's data protocol specification](https://docs.airbyte.io/architecture/airbyte-specification). 
+
+## 3. First-time contributors, welcome!
+
+We appreciate first time contributors and we are happy to assist you in getting started. In case of questions, just reach out to us via [email](mailto:hey@airbyte.io) or [Slack](https://slack.airbyte.io)!
 
 Here is a list of easy [good first issues](https://github.com/airbytehq/airbyte/labels/good%20first%20issue) to do.
 
-As soon as you are done with your development, just put up a PR.  
-When we review we look at:
+## 4. Areas for contributing
 
-* ‌Does the PR solve the issue?
-* Is the proposed solution reasonable?
-* Is it tested? \(unit tests or integration tests\)
-* Is it introducing security risks?
-* Does it pass QA?
+### **New integrations**
 
-‌Once your PR passes, we will merge it.
+It's easy to add your own integrations to Airbyte! Here are some links to instruct how to add sources and destinations. 
```

Comment:
```suggestion
It's easy to add your own integrations to Airbyte! Since Airbyte Connectors are encapsulated within Docker containers, you can use any language you like. Here are some links on how to add sources and destinations. 
```